docs/src/content/docs/zh-cn/reference/configuration.mdx

```diff
@@ -509,6 +509,13 @@ starlight({
 默认情况下，每个页面的 `<title>` 都是 `页面标题 | 网站标题`。
 举例，本页面的标题是“配置参考”，本站点的标题是“Starlight”，所以本页面的 `<title>` 是“配置参考 | Starlight”。
 
+### `disable404Route`
+
+**类型：** `boolean`  
+**默认值：** `false`
+
+禁用注入 Starlight 的默认 [404 页面](https://docs.astro.build/zh-cn/core-concepts/astro-pages/#自定义-404-错误页面)。要在项目中使用自定义的 `src/pages/404.astro` 路由，请将此选项设置为 `false`。
```

Comment:
```suggestion
禁用注入 Starlight 的默认 [404 页面](https://docs.astro.build/zh-cn/core-concepts/astro-pages/#自定义-404-错误页面)。要在项目中使用自定义的 `src/pages/404.astro` 路由，请将此选项设置为 `true`。
```

Shouldn't this be `true`? @delucis

Reply:
Oh, I think you are right. I just noticed the same issue while reviewing the French translation and opened [a PR](https://github.com/withastro/starlight/pull/1431)